active/DateTimePicker/DateTimePicker-Visual-Updates-Spec.md

```diff
@@ -0,0 +1,150 @@
+
+This spec will detail a number of changes to update the designs of DatePicker and TimePicker so that they are more 
+visually aligned with modern WinUI controls. At a high level, some key parts of these updated styles are: a rounded selection rectangle; rounded backplates on items, buttons, and scroll buttons on  hover; and new animations that are aligned with other controls. 
+
+There is only one major API change that will come with these styling updates, which is adding a new class called `ColorFilterOverlayControl`. This class provides the logic behind displaying the selection rectangle that spans across all three rows in DatePicker and TimePicker, and giving it the correct background color. 
+
+# Visual Examples
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+**DatePicker:**
+
+As you can see on the left, the old version of DatePicker contains squared elements inside of it. The new version of DatePicker will round off these elements.
+
+![Shows the old DatePicker visuals alongside the new DatePicker visuals](images/datepicker-evolution.PNG)
+
+**TimePicker:**
+
+Similarly to DatePicker, the old version of TimePicker contains squared elements inside of it. The new version of TimePicker will round off these elements.
+
+![Shows the old TimePicker visuals alongside the new TimePicker visuals](images/timepicker-evolution.PNG)
+
+**Rounded backplates and focus visuals**
+
+As you can see in the mockup below, all buttons and items (individual cells) within DatePicker and TimePicker will receive a rounded grey backplate when hovered or pressed. In addition to this, columns within DatePicker and TimePicker will receive rounded focus rectangles (as shown below).
+
+![Shows a DatePicker with scroll and check buttons being hovered, as well as the last column being focused upon](images/DPTP-other-states.PNG)
+
+# API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+## New APIs
+
+`public Class ColorFilterOverlayControl`
+
+Represents a class that takes a rectangle and transforms into the coordinate space of a TargetElement, and then runs a ColorMatrixEffect on that region to select the alpha channel and replace with a ReplacementColor.
```

Comment:
I don't know how to phrase this better but I'll give some other words in the hopes that someone can help edit this into documentation-appropriate terms:

This class assists in doing a specialized color filter applied to another element (the TargetElement). Using whatever rectangle of the ColorFilterOverlayControl overlaps the TargetElement, the ColorFilterOverlayControl renders that region of the TargetElement to an offscreen image and passes that image through a ColorMatrixEffect which takes the Alpha channel for each pixel of the TargetElement and replaces the RGB channels with the RGB of the ReplacementColor.

Reply:
@jevansaks How about this?

Represents a class that assists in applying a specialized color filter onto a TargetElement. The region of the TargetElement that has overlap with the ColorFilterOverlayControl is rendered to an offscreen image that's passed through a ColorMatrixEffect, which takes the Alpha channel of each pixel in the region. RGB channels are then replaced with the RGB of a ReplacementColor.

Jevan/@MikeHillberg Is this too long for a general description that'll be used in Intellisense? Perhaps only the first sentence should be use for Intellisense and the rest should be on the API reference page to provide more context.

Reply:
That description sounds good to me. For intellisense usually the first sentence of the description is used in the tooltip, and I think the first sentence you have is fine.